Directory.Build.props

```diff
@@ -4,29 +4,28 @@
      Adding Sdk="Microsoft.NET.Sdk" to a project causes NuGet files to be generated very early on, so we need these defines to be here
      to ensure that all of our build output files end up in the same location.  This is parsed too late when put in mux.controls.props. -->
 <Project InitialTargets="DirectoryBuildPropsInfo" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup Condition="$(BuildingWithBuildExe) != 'true'">
-    <IsTDPConfiguration>false</IsTDPConfiguration>
-
-    <BaseOutputPath>$(MSBuildThisFileDirectory)BuildOutput\$(Configuration)</BaseOutputPath>
-    <OutDir>$(BaseOutputPath)\$(Platform)\</OutDir>
 
-    <!-- For native projects that use Win32 to mean x86, override the OutDir
-         property to provide consistency between projects. -->
-    <OutDir Condition="'$(Platform)' == 'Win32'">$(BaseOutputPath)\x86\</OutDir>
+  <!-- Due to import ordering inconsistencies between different project types, we import this file from multiple places. -->
+  <Import Project="$(MSBuildThisFileDirectory)build\EnsureOutputLayout.props" Condition="$(EnsureOutputLayoutPropsIncluded) == ''" />
 
-    <OutputPath>$(OutDir)\$(MSBuildProjectName)\</OutputPath>
+  <PropertyGroup>    
+    <IsTDPConfiguration>false</IsTDPConfiguration>
 
     <GeneratedFilesDir>$(BaseIntermediateOutputPath)Generated Files\</GeneratedFilesDir>
     <GenerateProjectSpecificOutputFolder>True</GenerateProjectSpecificOutputFolder>
+
+    <CustomBeforeMicrosoftCommonTargets>$(BuildRoot)BeforeCommon.targets</CustomBeforeMicrosoftCommonTargets>
```

Comment:
><CustomBeforeMicrosoftCommonTargets>$(BuildRoot)BeforeCommon.targets</CustomBeforeMicrosoftCommonTargets> [](http://example.com/codeflow?start=4&length=105)

unless this is specified in command line, it's probably useless to define here (already too late). I think we can remove this alltogether.